src/test/test-packages/22-git-and-flux/zarf.yaml

```diff
@@ -0,0 +1,70 @@
+kind: ZarfPackageConfig
```

Comment:
@Racer159 @Noxsios 

I don't think I like what I did here but I want to get your thoughts on it. I didn't want to update an `example` with logic that was strictly for testing so I decided to make a test package. 

On the other hand, I didn't want to create a new test workflow that touched the cluster since that would be added test time when it isn't really necessary...